src.js

```diff
@@ -61,10 +61,34 @@ function isPalindrome(term) {
 	return false;
 }
 
+/**
+ * Returns an array of all the prime factors of a number.
+ * @param num
+ */
+function primeFactors(num) {
```

Comment:
Thanks Dan,

I was trying to minimize the amount of iterations that the loops would have to go through by getting rid all the two's that can divide into the factor, and then, in the next loop, you can count up by 2 instead of by 1 saving some time spent on checking the rest of the numbers divisible by 2. That being said, for simplicity sake, I like your code as it condenses it and would have saved coding time.

Additionally, I didn't realize that you could use let instead of var to create variables. I'm use to java and the only real way is to use int/String/etc. I'll keep it in mind next time I have a variable I'm setting equal to something.